### PR TITLE
Split RDataFrame_vs_Tree, update classes

### PR DESCRIPTION
**Pull request by HLilit**

### CODE REVIEW

File: root/tree/dataframe/RDataFrameBenchmarks.cxx [hunk 1]
```diff
@@ -135,49 +135,49 @@ static void BM_RDF_FillHistoRangeDeductionNoRDF(benchmark::State &state)
          h.Fill(i);
    }
 }
-BENCHMARK(BM_RDF_FillHistoRangeDeductionNoRDF)->Unit(benchmark::kMicrosecond)->Arg(100)->Arg(100000);
+BENCHMARK(BM_RDataFrame_FillHistoRangeDeductionNoRDF)->Unit(benchmark::kMicrosecond)->Arg(100)->Arg(100000);
 
 // Fill a histogram from a single thread
 // Data is generated on the fly, not read from a file
 // The overhead of generating the data and passing it to the Histo action should be
 // the one measured by BM_RDF_NoOpLoopOnCustomColumn
```
Comment (vgvassilev) [inline root/tree/dataframe/RDataFrameBenchmarks.cxx]: Could you update this comment to the renamed bm.

File: root/tree/tree/TTreeBenchmarks.cxx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#include "../dataframe/RDataFrameBench.h"
```
Comment (vgvassilev) [inline root/tree/tree/TTreeBenchmarks.cxx]: Is this some private header? Why do we need to include it that way

File: root/tree/dataframe/RDataFrameBenchmarks.cxx (hunk 1 above)
Comment (HLilit) [reply root/tree/dataframe/RDataFrameBenchmarks.cxx]: Done!

File: root/tree/tree/TTreeBenchmarks.cxx (hunk 2 above)
Comment (HLilit) [reply root/tree/tree/TTreeBenchmarks.cxx]: Done!

**Comment by oshadura** [comment]
@vgvassilev I will merge, since I want start to collect data. I think Lilit resolved all your remarks.

**Pull request merged**